### ISSUE TITLE
Not working on windows

### ISSUE THREAD

**Issue by felipecrrmts** [gowebly/gowebly#6]
**Required check list:**

- [x] I didn't find in the repository's issues section a similar bug.
- [x] I understand, this is an Open Source and not-for-profit product.
- [x] This is not about a third-party project, framework, or technology.

**My environment:**

- OS (`MINGW64_NT-10.0-22621 DND 3.4.7-25de8b84.x86_64 2023-08-28 21:32 UTC x86_64 Msys`):
- Go (`go1.21.1 windows/amd64`):
- gowebly (`latest`):

**Describe the bug:**
when running on windows error happens

```
                          _     _
   __ _  _____      _____| |__ | |_   _
  / _` |/ _ \ \ /\ / / _ \ '_ \| | | | |
 | (_| | (_) \ V  V /  __/ |_) | | |_| |
  \__, |\___/ \_/\_/ \___|_.__/|_|\__, |
  |___/ build amazing Go web apps |___/

✕ Please fix error(s):

 – gowebly: run without any commands or/and options

‼ For more information, see https://gowebly.org
```

runned on cmd, bash and powershell.
my workaround was running inside wsl2/ubuntu

**Comment by koddr** [comment]
Hi @felipecrrmts 

And what command do you enter? This error tells you that none of the available commands were defined.

Try starting with this one:

```console
gowebly create
```

**Comment by felipecrrmts** [comment]
both have the same outcome

**Comment by koddr** [comment]
What do you mean by "the same outcome"? All the tests I did on Windows 10 and 11 (via WSL 2, of course) resulted in satisfactory results. The `gowebly` CLI works exactly as intended.

Can you be more concrete, please? 🤔 

Otherwise, I will not be able to help you to understand the problem of your system.

**Comment by koddr** [comment]
Oh, I hear you.

Can you please try to install the latest `gowebly` CLI v1.3.0 directly (on your Windows system, without WSL) via the native Go command like this:

```console
go install github.com/gowebly/gowebly@latest
```

And run it (into the empty folder):

```console
gowebly create
```

Will an error occur when doing so? 🤔 Please send here what your console outputs.

**Comment by felipecrrmts** [comment]
`dnd@DND MINGW64 ~
$ go install github.com/gowebly/gowebly@latest
go: downloading github.com/gowebly/gowebly v1.3.0

dnd@DND MINGW64 ~
$ gowebly create
                          _     _
   __ _  _____      _____| |__ | |_   _
  / _` |/ _ \ \ /\ / / _ \ '_ \| | | | |
 | (_| | (_) \ V  V /  __/ |_) | | |_| |
  \__, |\___/ \_/\_/ \___|_.__/|_|\__, |
  |___/ build amazing Go web apps |___/

○ Preparing the backend part... Please wait!

✕ Please fix error(s):

 – template: pattern matches no files: `templates\backend\default\go.mod.tmpl`

‼ For more information, see https://gowebly.org`

**Comment by koddr** [comment]
Hi, @felipecrrmts 

I think I've figured out what's going on here. 

When files and folders are embedded by the `//go:embed folder/*` directive in a Go program, their path becomes Unix-style (i.e., `folder/file.txt`). But I thought it should be in the style of the OS on which the program is running (i.e., on Windows, the path should be backslash `folder\file.txt`). It wasn't.

Can you please try this commit [4f41624](https://github.com/gowebly/gowebly/pull/9/commits/4f4162437b06114136d8b5b2f1b516ca2dae42e3) on your Windows system and send here what your console outputs?

**Comment by koddr** [comment]
Fixed in `v1.3.1`.

@felipecrrmts please try this release. I'll close this issue, but if something goes wrong, feel free to re-open it and describe a new problem (if one occurs).

**Comment by felipecrrmts** [comment]
working now, amazing job 💯
